### PR TITLE
refactor: use spread operator replace object-assign

### PR DESCRIPTION
**Pull request by paranoidjk**
- Upgrade `omit.js` to v1.0, which was build by rc-tools new version with babel-plugin-transform-assign

- Upgrade `antd-tools` to new version, which include babel-plugin-transform-assign too

- remove `object-assign`, prefer to use spread operator, or Object.assign
- - -

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

**Comment by benjycui** [comment]
pending

Waiting for antd-mobile to make sure that it will not affect server rendering https://github.com/ant-design/antd-mobile-samples/issues/17

**Comment by afc163** [comment]
conflict

**Comment by paranoidjk** [comment]
I test [nextjs-with-antd-mobile](https://github.com/zeit/next.js/tree/v3-beta/examples/with-antd-mobile) in safari and chrome, there is no checksum warning, so i think we should merge this, in case get more conflict code in future.

Btw, thanks for @cncolder great job. 👍

File: components/date-picker/RangePicker.tsx [hunk 1]
```diff
@@ -249,7 +249,13 @@ export default class RangePicker extends React.Component<any, any> {
     };
 
     return (
-      <span className={classNames(props.className, props.pickerClass)} style={assign({}, style, pickerStyle)}>
+      <span
+        className={classNames(props.className, props.pickerClass)}
+        style={{
+        ...style,
```
Comment (benjycui) [inline components/date-picker/RangePicker.tsx]: 缩进。

**Comment by benjycui** [comment]
没其它问题了，改完缩进可以合并。

**Comment by paranoidjk** [comment]
@benjycui Updated. https://github.com/ant-design/ant-design/pull/6677/commits/20a73ec44ab05fb9dd07fe5692d23b5d0c6e58a4

**Pull request merged**